resolwe_bio/processes/reads_processing/hsqutils.yml

```diff
@@ -0,0 +1,86 @@
+---
+- slug: hsqutils-trim
+  name: HSQutils trim
+  requirements:
+    expression-engine: jinja
+  data_name: '{{ reads.fastq.0.file|default("?") }}'
+  version: 0.0.1
+  type: data:reads:fastq:paired:hsqutils
+  category: analyses
+  flow_collection: sample
+  persistence: CACHED
+  description: >
+    The HSQutils trim tool is designed to remove all HEAT-Seq primer bases and UID bases from the reads prior to mapping.
+  input:
+    - name: reads
+      label: NGS reads
+      type: data:reads:fastq:paired
+    - name: probe
+      label: Probe information file
+      type: data:file
+  output:
+    - name: fastq
+      label: Reads file (mate 1)
+      type: list:basic:file
+    - name: fastq2
+      label: Reads file (mate 2)
+      type: list:basic:file
+    - name: fastqc_url
+      label: Quality control with FastQC (Upstream)
+      type: list:basic:file:html
+    - name: fastqc_url2
+      label: Quality control with FastQC (Downstream)
+      type: list:basic:file:html
+    - name: fastqc_archive
+      label: Download FastQC archive (Upstream)
+      type: list:basic:file
+    - name: fastqc_archive2
+      label: Download FastQC archive (Downstream)
+      type: list:basic:file
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      NAME1=`basename '{{ reads.fastq.0.file }}' .fastq.gz`
+      NAME2=`basename '{{ reads.fastq2.0.file }}' .fastq.gz`
+
+      {% for r in reads.fastq %}
```

Comment:
Use this instead of for loop:

    FW_READS="{{ reads.fastq|join(' ') }}"

Fix this in whole PR.